be/src/vec/columns/column_const.cpp

```diff
@@ -209,4 +209,11 @@ std::pair<ColumnPtr, size_t> check_column_const_set_readability(const IColumn& c
     }
     return result;
 }
+
+const ColumnPtr& unpack_if_const(const ColumnPtr& ptr) noexcept {
```

Comment:
better return std::pair<bool, ColumnPtr&> to reduce unless virtual function call